docs/source/markdown/podman-ps.1.md

```diff
@@ -32,12 +32,19 @@ all the containers information.  By default it lists:
 
 **--all**, **-a**
 
-Show all the containers, default is only running containers
+Show all the containers including those created by other container technologies such as Buildah and CRI-O.  The default displays only running containers created by Podman.  Containers not created by Podman have an 'NA' status.
 
 **--pod**, **-p**
 
 Display the pods the containers are associated with
 
+**--storage**
+
+Display containers that are not controlled by Podman but are stored in containers storage.  These containers
+are generally created via other container technology such as Buildah or CRI-O and may depend on the
+same container images that Podman is using.  These containers are denoted with a 'NA' in the COMMAND and
```

Comment:
Does the 'NA' still apply?  Your example shows 'Buildah`

Reply:
Nope, good catch.